region/maridia/outer/Boyon Gate Hall.json

```
@@ -1398,6 +1398,47 @@
       "farmCycleDrops": [{"enemy": "Zebbo", "count": 1}],
       "flashSuitChecked": true
     },
+    {
+      "link": [4, 4],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit)",
+      "requires": [
+        {"obstaclesCleared": ["R-Mode"]},
+        {"refill": ["Energy"]},
+        {"or": [
+          {"and": [
+            {"obstaclesCleared": ["A"]},
+            {"canShineCharge": {"usedTiles": 15, "openEnd": 1}}
+          ]},
+          {"and": [
+            {"doorUnlockedAtNode": 1},
+            {"doorUnlockedAtNode": 2},
+            {"canShineCharge": {"usedTiles": 14, "openEnd": 0}}
+          ]},
+          {"and": [
+            {"or": [
+              {"doorUnlockedAtNode": 1},
+              {"doorUnlockedAtNode": 2}
+            ]},
+            {"canShineCharge": {"usedTiles": 13, "openEnd": 0}}
+          ]},
+          {"canShineCharge": {"usedTiles": 12, "openEnd": 0}}
+        ]},
+        {"autoReserveTrigger": {"maxReserveEnergy": 95}},
+        "canRModeSparkInterrupt"
+      ],
+      "unlocksDoors": [
+        {"nodeId": 1, "types": ["ammo"], "requires": []},
+        {"nodeId": 2, "types": ["ammo"], "requires": []}
+      ],
+      "resetsObstacles": ["R-Mode"],
+      "flashSuitChecked": true,
+      "blueSuitChecked": true,
+      "note": [
+        "Fill up on the respawning Zebbos.",
+        "Open the gate to use that runway, or else run between the bottom doors and quickly walljump",
+        "back up to the Boyon."
```

You forgot to include ways to get up in the code. Probably hijump spring would also work.
```suggestion
        "Fill up on the respawning Zebbos. Use the runway with the gate or at the bottom of the room."
```